Error when overriding user manager

# Bug report

I'm overwritting the user manager like:

```python
class MyUserManager(BaseUserManager):
    def create_user(
        self,
        facebook_email: str,
        facebook_first_name: str,
        facebook_last_name: str,
        facebook_uid: str,
        facebook_access_token: str,
        **extra_fields
    ):
    .....


class User(AbstractBaseUser):
    (...)
    objects = MyUserManager()
```
## What's wrong

mypy returns:

```
accounts/models.py:31: error: Missing type parameters for generic type "BaseUserManager"

```
## How is that should be

It should just work :) 

## System information

- OS: Ubuntu 18.04
- `python` version: 3.6
- `django` version: 2.2
- `mypy` version: 0.761
- `django-stubs` version: 1.4.0

--- a/tests/typecheck/models/test_contrib_models.yml
+++ b/tests/typecheck/models/test_contrib_models.yml
@@ -26,3 +26,22 @@
 
         from django.contrib.auth.models import Group
         reveal_type(Group().name)  # N: Revealed type is "builtins.str"
+
+-   case: can_override_abstract_user_manager
+    main: |
+        from myapp.models import User
+        reveal_type(User.objects)  # N: Revealed type is "myapp.models.UserManager[myapp.models.User]"
+        reveal_type(User.objects.all())  # N: Revealed type is "django.db.models.query._QuerySet[myapp.models.User, myapp.models.User]"
+    installed_apps:
+        - django.contrib.auth
+        - myapp
+    files:
+        - path: myapp/__init__.py
+        - path: myapp/models.py
+          content: |
+              from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
+              class UserManager(BaseUserManager["User"]):
+                  ...
+
+              class User(AbstractBaseUser):
+                  objects = UserManager()

